posts/how-to-install-sourcegraph-on-linux-with-docker.md

```diff
@@ -25,15 +25,15 @@ On a Linux cloud server or local computer, you can install Docker via the comman
 
 Ensure that you have the [snap package manager](https://snapcraft.io/) installed by running the following commands in your terminal for **Ubuntu or Debian Linux**:
 
-```
+```bash
```

Comment:
I noticed I hadn't made rendered these as bash the first time around, fixing now

Reply:
I notice we use `sh` in the README (looks like that convention was first introduced [here](https://github.com/sourcegraph/learn/commit/12684d8be7a8bdee3b340dc002cc568966621f63)). Personally, I don't care what we use as our "default", but I guess we should stay consistent. Do you have any preferences?

Reply:
Ah, I can update to `sh`